.buildkite/pipeline.yml

```diff
@@ -387,6 +387,15 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: ":construction: Moist Held Suarez"
+        key: "moist_held_suarez"
+        command:
+          - sed 's/t_end = "1000days"/t_end = "1days"/' experiments/ClimaEarth/run_moist_held_suarez.jl > experiments/ClimaEarth/run_moist_held_suarez_modified.jl
```

Comment:
how are the modified drivers different from the unmodified ones?

Reply:
this is just changing the length of the simulation for CI (so it doesn't run for 1000d like in the hierarchy pipeline). There will be more elegant ways of doing it, but it works for our purposes here.

Reply:
could we rename it from `_modified` to `_short` to be more descriptive?

Reply:
of course, good point!